embedding/cpp/flutter_engine.cc

```diff
@@ -79,6 +79,7 @@ std::vector<std::string> ParseEngineArgs() {
     auto engine_args_it =
         std::find(engine_args.begin(), engine_args.end(), "--enable-impeller");
     if (engine_args_it == engine_args.end() && it->second == "true") {
+      *is_impeller_enabled = true;
```

Comment:
In case of (engine_args_it != engine_args.end() && it->second == "true"), *is_impeller_enabled is also true, right?